go-controller/pkg/ovn/master.go

```diff
@@ -765,20 +769,50 @@ func (oc *DefaultNetworkController) addUpdateLocalNodeEvent(node *kapi.Node, nSy
 		}
 	}
 
+	if nSyncs.syncZoneIc && config.OVNKubernetesFeature.EnableInterconnect {
```

Comment:
I normally tend to be over cautious when access pointers and not assume caller has done the checks.  Since oc.zoneChassisHandler would be null if EnableInterconnect is false,  checking the flag to be sure.